src/main/java/propertyFiles/Bundle.properties

```diff
@@ -1934,7 +1934,7 @@ file.downloadBtn.format.all=All File Formats + Information
 file.downloadBtn.format.tab=Tab-Delimited
 file.downloadBtn.format.original={0} (Original File Format)
 file.downloadBtn.format.rdata=RData
-file.downloadBtn.format.var=Variable Metadata
+file.downloadBtn.format.var=DDI Codebook v2
```

Comment:
I'm happy to see this cleanup, this renaming to "DDI Codebook v2". Can we mention it in a release note (along with the feature in general)?

Reply:
> I'm happy to see this cleanup, this renaming to "DDI Codebook v2". Can we mention it in a release note (along with the feature in general)?

I'm generally very pro-documentation, and usually an eager writer of release notes... But this feels like a case where I'm inclined to push back. There is no "feature" really - this would honestly be like adding a release note on having fixed a typo in the guide.  This PR is addressing a minor inconsistency, something we potentially know about a file that we weren't exporting; but it's worth noting that nobody has ever asked for it - it was just something that Julian noticed (6 years ago, lol). Same for editing the UI labels - I just clarified that the DDI format we produce is in fact DDI version such and such. Doesn't seem like release note material - ?

It also feels like it I went over the initially allocated "size 3" just writing this paragraph. :)

Reply:
Fair enough. It'll be a nice surprise for people who are really paying attention. 😄